restlicsharpclient/restliclienttest/UrlParamUtilTests.cs

```diff
@@ -0,0 +1,108 @@
+﻿/*
+   Copyright (c) 2017 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+using System;
+using System.Collections.Generic;
+
+using restlicsharpclient.restliclient.util;
+
+namespace restlicsharpclient.restliclienttest
+{
+    [TestClass]
+    public class UrlParamUtilTests
```

Comment:
1. For completeness, you can refer to http://jarvis.corp.linkedin.com/codesearch/result/?path=restli-client-lib%2FRestLiClientLibTests&reponame=si-pegasus%2Frestli-client-lib&name=TestURLBuildingFromSwift.swift for all tests on this class.

2. Also, we may need test for RequestUrlBuilder.